src/main/java/frc/robot/mechanisms/pivot/Pivot.java

```diff
@@ -62,11 +64,12 @@ public void periodic() {}
     /**
      * Sets the intake motor to a specified position.
      *
-     * @param percent percentage of max rotation (0 is vertical). Note that the percentage is not
-     *     [-1,1] but rather [-100,100]
+     * @param angle angle of max degrees (70 is max). Note that the angle measure will be between 0
+     *     and 70
      */
-    public Command runPosition(double percent) {
-        return run(() -> setMMPosition(percentToRotation(percent))).withName("Pivot.runPercent");
+    public Command runPosition(double angle) {
+        return run(() -> setMMPosition(percentToRotation((70 - angle) / 70 * 100)))
```

Comment:
let's get rid of everything percent related. ie. refactor the method from percentToRotation to angleToRotations() and move the math happening in the method parameters into the method itself

also, remember that we always want constants as variables. we should move maxAngle to the pivot constants and reference it here instead of writing 70 (this gets important when our robots have different max angles, which they do)